### DISCUSSION TITLE
No audio on service start up

### DISCUSSION THREAD

**Discussion by Darbstar** [mikebrady/shairport-sync#1777] [Q&A]
Hi all.  Probably fixed, sorry I didn't see.
Raspberry Pi Zero 2 with Hifiberry Amp2 on tower speakers.  Used Lite OS, headless.

Followed instructions: https://github.com/mikebrady/shairport-sync/blob/master/BUILD.md

used hifiberry instructions: https://www.hifiberry.com/docs/software/configuring-linux-3-18-x/
 
the amp reads as hw=0 I believe.  When logged in with SSH and I start the shairport, system is good and audio confirmed.

ERROR:
When I enabled the service on start up, (as per instructions, 'sudo systemctl enable ....'  the device is seen on airplay but no audio comes out. 

Followed instructions from Troubleshooting: https://github.com/mikebrady/shairport-sync/blob/master/TROUBLESHOOTING.md
I have disabled the service (sudo systemctl disable...', added the timer script , and even increased the timing to 30s (no joy).

Next: 
Logged in (SSH) while service running and no audio.   Command:  sudo systemctl status shairport-sync

Found the error is 'the output device "default" is busy and can't be used by Shairport Sync at present.

Next:
I have modified the shairport-sync.conf to make the default backend the name device in aplay (no joy)

Any feedback on the failure?  Thank you all for your help and patience.  This is actually my first pi project and loving the challenges.  This is just the first wall I don't know where to start to tackle  
![alsamixer](https://github.com/mikebrady/shairport-sync/assets/154937879/0f7ccc2a-fe8c-482b-b37c-3835455df8c3)
![enable](https://github.com/mikebrady/shairport-sync/assets/154937879/6be940d1-3663-4785-822f-64885d4e972e)
![asound](https://github.com/mikebrady/shairport-sync/assets/154937879/cc6440c1-9236-40d1-bebe-e3fcc0cccdca)
![shairport-sync conf](https://github.com/mikebrady/shairport-sync/assets/154937879/880038fd-bdbb-4bbc-a9df-f44179b8d319)
![aplay](https://github.com/mikebrady/shairport-sync/assets/154937879/2740ba76-24f0-4b04-8050-54fea97d6e31)

**Comment by mikebrady** [comment]
This has been dealt with [here](https://github.com/mikebrady/shairport-sync/issues/1778).
